python/cugraph/community/ktruss_subgraph.py

```diff
@@ -13,6 +13,43 @@
 
 from cugraph.community import ktruss_subgraph_wrapper
 from cugraph.structure.graph import Graph
+from cugraph.utilities import is_networkx_graph
+from cugraph.utilities import convert_from_nx
+from cugraph.utilities import cugraph_to_nx
+
+
+def k_truss(G, k):
+    """
+    Returns the K-Truss subgraph of a graph for a specific k.
+
+    This API is for NwtworkX compatability.
+
+    Parameters
+    ----------
+    G : cuGraph.Graph or networkx.Graph
+        cuGraph graph descriptor with connectivity information. k-Trusses are
+        defined for only undirected graphs as they are defined for
+        undirected triangle in a graph.
+
+    k : int
+        The desired k to be used for extracting the k-truss subgraph.
+
+    Returns
+    -------
+    G_truss : cuGraph.Graph or networkx.Graph
+        A cugraph graph descriptor with the k-truss subgraph for the given k.
+        The networkx graph will NOT have all attributes copied over
+    """
+
+    is_nx = is_networkx_graph(G)
```

Comment:
would it be better to use check_nx_graph to be consistent?

Reply:
the issue was that I needed to keep a reference to the original input Graph. But I could uuse nx_check_graph and just update variable names